src/chainparams.cpp

```diff
@@ -194,10 +194,11 @@ class CMainParams : public CChainParams {
             ( 407452, uint256S("0x000000000003c6a87e73623b9d70af7cd908ae22fee466063e4ffc20be1d2dbc"))
             ( 523412, uint256S("0x000000000000e54f036576a10597e0e42cc22a5159ce572f999c33975e121d4d"))
             ( 523930, uint256S("0x0000000000000bccdb11c2b1cfb0ecab452abf267d89b7f46eaf2d54ce6e652c")),
```

Comment:
pls remove `,` at the end of this line

Reply:
Fixed. Oh man...that struct and the initialization of it is horrible :D
But luckely I have a backported refactoring of this in my queue... :)